spec/features/friend_invitation_spec.rb

```diff
@@ -49,21 +41,18 @@ def expect_invitation_sent_to(email)
   end
 
   def follow_email_link_to_register(email)
-    open_email(email)
+    #open_email(email)
     current_email.click_link 'Register'
   end
 
   def register_invited_user
-    fill_in 'Password', with: 'password'
+    fill_in 'Password', with: 'wildhorses'
     fill_in 'Full Name', with: 'Invited User'
+    find("input[data-stripe='number']").set('4242424242424242')
+    find("input[data-stripe='cvc']").set('123')
+    select 'June', from: 'date_month'
+    select '2016', from: 'date_year'
     click_button 'Sign Up'
```

Comment:
The issue is actually here - Sign Up is ajaxified via Stripe.js and thus you have to tell Capybara to wait couple seconds for the response from Stripe server about the token returned.